server.go

```diff
@@ -3345,9 +3346,14 @@ func (s *Server) DiagnosticsAsRows() []*influxql.Row {
 
 		// Shard may not be local to this node.
 		if sh.store != nil {
-			shardsRow.Columns = append(shardsRow.Columns, "path")
-			shardsRow.Values[0] = append(shardsRow.Values[0], sh.store.Path())
+			if !hasPath {
```

Comment:
I don't think this is necessary. If a `sh.store` is non-nil, then it must have a path.